scripts/fonts.json

```diff
@@ -120,15 +124,14 @@
     ],
     "khmer": [[6016, 6911]],
     "latin": [
-      [33, 767],
+      [33, 879],
       [7424, 7615],
       [7680, 7935],
-      [8304, 8591],
+      [8192, 8703],
       [11360, 11391],
       [42784, 43007],
       [43824, 43887],
-      [64256, 64335],
-      [65280, 65519],
+      [64256, 64271],
```

Comment:
Reduce [Alphabetic Presentation Forms](https://en.wikipedia.org/wiki/Alphabetic_Presentation_Forms) to just the Latin part

Move fullwidth to its own category to better match CJK text